src/main/kotlin/org/rust/ide/navigation/goto/RsGotoTargetRendererProvider.kt

```diff
@@ -20,8 +21,11 @@ class RsGotoTargetRendererProvider : GotoTargetRendererProvider {
         if (!gotoData.hasDifferentNames() && targetsInImpl) {
             return GoToImplRenderer()
         }
-        if (element is RsImplItem) return ImplRenderer()
-        return null
+        return when (element) {
+            is RsImplItem,
+            is RsAbstractable -> NamedElementRenderer()
+            else -> null
+        }
```

Comment:
This change causes ClassCastException when typing `Ctrl + Alt + B` on a trait.

Also, I don't understand why this is needed because we already have fine rendering for 
`Ctrl + Alt + B` implementation variants
![image](https://user-images.githubusercontent.com/3221931/86228759-71af6f80-bb97-11ea-8df9-c3ed0ccbd89e.png)

Reply:
You're right, this doesn't seem to be needed.